sdks/java/extensions/sql/zetasql/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
   compile project(":sdks:java:core")
   compile project(":sdks:java:extensions:sql")
   compile library.java.vendored_calcite_1_20_0
+  compile library.java.guava
```

Comment:
Here we may need to diverge eventually. ZetaSQL is a Google "throw over the wall" open source. That means that it generally will track against the latest Guava or even newer internal changes. Just a note - if we hit any problems it may simply have to diverge.

Reply:
diverging will be bad for the project as its unlikely that this won't cause various IOs/runners to fail

Reply:
I was just pointing out that if there comes a time when there is no version of Guava that works for both ZetaSQL and Kinesis we might have no choice. Neither one can be vendored. ZetaSQL *might* be possible to vendor by putting a layer between user-supplied parameter values for prepared statements.